tensorflow/core/tpu/tpu_initializer_helper.cc

```diff
@@ -261,13 +254,20 @@ bool FindAndLoadTpuLibrary() {
   if (library) {
     // We can open the shared library which means we are in a TPU environment.
     // Try to acquire exclusive access.
-    if (TryAcquireTpuLock()) {
-      InitializeTpuLibrary(library);
+    Status tpu_lock_status = TryAcquireTpuLock();
```

Comment:
Use TF_RETURN_IF_ERROR instead of manually checking the status (check codesearch for examples)